### DISCUSSION TITLE
Using Syncthing to Sync Feeder

### DISCUSSION THREAD

**Discussion by wynteryyc** [spacecowboy/Feeder#362] [Q&A]
Hi all,

I use Syncthing to sync everything basically. Is it possible to get at the files (on Android) directly? I'd like to be able to add a new Sync chain on Syncthing to just sync the files/data that Feeder has on the Android file system. Hope that question makes sense. Basically, I'm asking what the files are that contain all the feeds and all that, where they are (by default) and if copying them using Syncthing will break Syncthing on a second device?

TIA

**Comment by spacecowboy** [comment]
No it's not.

Newer versions of Android has changed how files are handled. They are only (reliably) accessible by the app itself.

To expose them to other apps would require a ContentProvider, and SyncThing doesn't support that on their end.

**Comment by spacecowboy** [comment]
Also, copying database etc is not supported and never will be

**Comment by wynteryyc** [comment]
ok thanks!

Dale Forrest
https://sacredpeak.ink

Sent from Proton Mail Android

-------- Original Message --------
